### ISSUE TITLE
Renaming packages in Atmosphere

### ISSUE THREAD

**Issue by Urigo** [percolatestudio/atmosphere#241]
I'm not talking about renaming for Meteor compatibility :)
If I have a package named urigo:thebestpackage and I want to rename it to urigo:not-so-best-package 
but I don't want it to show as 2 different packages in atmosphere and that the downloads number will be the same.
Thanks

**Comment by domyen** [comment]
As far as we know, Meteor Packaging doesn't support this. Hopefully in the future?

I suppose an un-ideal way to do it is to hide the existing package and republish with a different name, however the download count and stats will be lost. :(

**Comment by tmeasday** [comment]
It's definitely coming from core. Not sure when though. 

Can we do anything simple that helps here? I can't really think of anything that wouldn't be wasted effort when it comes downstream from MDG.

**Comment by zimme** [comment]
I'm about to rename my `zimme:iron-router-active` package to `zimme:active-route` because I'm working on adding support for `meteorhacks:flow-router`. With support for another router I don't feel the name is representing the package functionality.

Can you guy's help me with the transition? Keep the history at atmosphere and such.

What I'm going to do is this:
1. Publish `zimme:active-route` at a version `2.0.0`, as I'm keeping and renaming the  current Github repo.
2. Publish a new version of `zimme:iron-router-active` from a separate branch, which will
   use and imply `zimme:active-route` and change description to say it's deprecated. See packages in note*\* too see what this looks like.
3. Have you guys transfer all the history events from `zimme:iron-router-active` to `zimme:active-route`. I assume it's not possible with number of downloads as you probably get those from the meteor packages server or something. I don't think that matters as when I use and imply `zimme:active-route` downloads will start to count for that package, otherwise it will rise again once people starts changing `zimme:iron-router-active` to `zimme:active-route` in their projects.

*\* I've done this with [`zimme:vide`](https://atmospherejs.com/zimme/vide) and [`zimme:bootstrap-growl`](https://atmospherejs.com/zimme/bootstrap-growl) because I've managed to get meteor packaging into core library repo. These packages weren't as popular as `zimme:iron-router-active` so I didn't care to transfer the history and such.

**Comment by zimme** [comment]
After looking at the history log at https://atmospherejs.com/zimme/iron-router-active I realize there isn't really any history events for this package other then the download count thing... So I might not need your help with anything.

However, it would be nice to have a better flow for renaming packages. :+1:

**Comment by tmeasday** [comment]
Yeah. I guess it's fairly rare for people to rename packages (apart from the old `mrt:X` packages from the Meteorite transition). I guess if we had official support for it we might look into bringing over the top-Xs and stuff..

**Comment by zimme** [comment]
@tmeasday, Sorry to bother you but I've taken this, MeteorCommunity/discussions#72, initiative. It would result in a need for renaming/moving packages where the metadata around the packages is quite important.

Do you have any thoughts on how we can make renaming/moving packages as smooth as possible.

Otherwise, maybe you know who at MDG would be best to ping into this issue. I'm thinking ekate but don't wanna ping core devs unnecessarily.

**Comment by tmeasday** [comment]
I think @ekatek is the person to ask. Let's confine this discussion to https://github.com/meteor/meteor/issues/2392 to avoid confusion.
